blockchain/chain_sync/src/chain_muxer.rs

```diff
@@ -161,7 +161,7 @@ pub struct ChainMuxer<DB, M, C: Consensus> {
 
 impl<DB, M, C> ChainMuxer<DB, M, C>
 where
-    DB: BlockStore + Sync + Send + 'static,
+    DB: BlockStore + Unpin + Sync + Send + 'static,
```

Comment:
Why is `Unpin` needed?

Reply:
For `DerefMut` trait

Reply:
How is `Unpin` related to `DerefMut`?

Reply:
https://doc.rust-lang.org/std/pin/struct.Pin.html#impl-DerefMut

Reply:
Do we pin a blockstore anywhere?

Reply:
It's just a constraint we need to have, otherwise it doesn't build. Earlier this constraint was guaranteed by Arc